app/assets/javascripts/discourse/components/topic-timer-info.js.es6

```diff
@@ -18,6 +19,13 @@ export default Ember.Component.extend(
     buildBuffer(buffer) {
       if (!this.get("executeAt")) return;
 
+      const topicClosed = this.get("topicClosed");
+      if (
+        topicClosed !== undefined &&
+        (topicClosed ? "close" : "open") === this.get("statusType")
```

Comment:
This looks weird to me. Here's how I would have written it

```javascript
const status = this.get("topicClosed") ? "close" : "open";

if (this.get("statusType") === status) { return; }
```

Reply:
Good thinking, I will refactor/rename the variables to make the intent clearer! Reasoning behind the `undefined` check: https://github.com/discourse/discourse/pull/6123#discussion_r204109276